src/shared/components/bottomSheet/BottomSheetWrapper.tsx

```
@@ -0,0 +1,114 @@
+      />
+      <div
+        ref={sheetRef}
+        data-sheet="true"
+        className={clsx(
+          styles.sheetWrapper,
+          isOpen ? styles.sheetWrapperExpanded : styles.sheetWrapperCollapsed
+        )}
+        onTransitionEnd={onExited}
+        onClick={(e) => e.stopPropagation()}
+        onTouchStart={(e) => e.stopPropagation()}
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)